python/hail/context.py

```diff
@@ -158,7 +158,7 @@ def import_gen(self, path, sample_file=None, tolerance=0.02, npartitions=None, c
         >>> (hc.import_gen('data/example.gen', sample_file='data/example.sample')
         ...    .write('output/gen_example1.vds'))
 
-        Load multiple files at the same time with `Hadoop glob patterns <../reference.html#hadoopglob>`_:
+        Load multiple files at the same time with `Hadoop glob patterns <hadoop_glob_patterns.html>`_:
```

Comment:
I'm sorry I didn't catch this on the previous PR. Should be:

```
:ref:`Hadoop glob patterns <sec-hadoop-glob>`
```

Could you please fix in `import_vcf` as well?